js/modules/sliderOurVisitors.js

```diff
@@ -0,0 +1,82 @@
+const slidesContainer = document.getElementById("ourVisitors__photo_container");
+const slidesQuantity = slidesContainer.querySelectorAll(".ourVisitors__photo").length;
+const SLIDER_MAX_POSSIBLE_POSITION = 0;
+const SHIFT = 100;
```

Comment:
bad name, not enough information, maybe SLIDE_SHIFT_IN_PERCENT or something like this

Reply:
Done.